_posts/2020-04-27-introducing-command-mode.adoc

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: 'Introducing Command Mode'
+date: 2020-04-27
+tags: development-tips 
+author: maxandersen
+---
+
+Ever wanted to use Quarkus awesome API's and full feature set from a command line application ? 
+
+Did you need to run a scheduled batch job now and then and not wanting to embed it into your main Quarkus built service ? 
+
+image:https://i.imgflip.com/3xwzsh.jpg[]
+
+Quarkus has thus far been used to write applications that starts an endpoint (i.e. webserver via http) and then runs for a long time.
+
+In Quarkus 1.4 command mode let you write apps that runs without an endpoint and optionally exits immediatly.
```

Comment:
Explain that it's in preview somewhere.

Reply:
sorry - not following what you mean here ?

Reply:
Command mode is considered stable?

Reply:
@stuartwdouglas ^^wdyt ? any api changes planned/expected?

Reply:
Not unless it turns out there are major problems. There are improvements we may want to make (like automatic parameter processing) but the basic API should be stable.

Reply:
thats what I thought - the api there is now is minimal and works. Future would be extensions/addons not changes thus I wouldn't call this preview unless @emmanuelbernard 
 for some reason want to play it extremely safe ? :)

Reply:
alright alright